### DISCUSSION TITLE
/vd on firefox is much better (imho) with browser.tabs.insertAfterCurrent=true (default is false)

### DISCUSSION THREAD

**Discussion by codaroma** [arnoudkooi/SN-Utils#500] [General]
title says it all

**Comment by arnoudkooi** [comment]
Looked it up, a bit of context (why/what/how) would help.

**Reply by codaroma** [reply]
For me, if I have the default value (false) and I have 20 tabs open, if say I do /vd on tab number 1 then the new tab gets opened as the very last tab (ie tab number 21). So it is hard to keep related tabs together. In this example the open record tab and the /vd tab are 20 tabs apart from each other. But if I change this option value to true, then the /vd tab opens directly adjacent to the tab that launched it. So in this example the /vd tab would be tab number 2 (right next to tab number 1).

**Reply by codaroma** [reply]
There are other browser addons that can manage the position of new tabs. But this setting works for me so far.

**Comment by Anduh** [comment]
@codaroma think this might have been fixed at one point, my Firefox don't need `browser.tabs.insertAfterCurrent`=`true`.

Firefox with v.9.2.0.0. `/vd` opens tab next to the record tab, not as last tab on list
- `about:config` show `browser.tabs.insertAfterCurrent`=`false` for me 
- Don't think my other extensions change this behavior.

(if fixed, you could close this discussion)
